cloudbuild-build-and-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+steps:
+  # Builds the Docker image
+  - name: docker
+    id: build_proxy_service_image
+    entrypoint: "sh"
+    args:
+      - "-c"
+      - |
+        echo "*************************************************************"
+        echo "* Building Docker image for environment...                  *"
+        echo "*************************************************************"
+        docker build -t "$_EQ_CIR_PROXY_SERVICE_IMAGE_REPO:${SHORT_SHA}" .
+
+  # Pushes the Docker image to Artifact Registry
+  - name: docker
+    id: push_proxy_service_image
+    entrypoint: "sh"
+    waitFor:
```

Comment:
These steps are all run in sequence, so don't think we'd need `waitFor` at the moment?